### ISSUE TITLE
Report generation fails when Cargo.toml binary target name contains a hyphen

### ISSUE THREAD

**Issue by aaronenberg** [taiki-e/cargo-llvm-cov#231]
cargo-llvm-cov v0.5.2
LLVM version 15.0.0-rust-1.65.0-

I've been using this version for a week or so and out of nowhere just yesterday I am seeing an error running `cargo llvm-cov report`, Specifically, I am running `cargo llvm-cov report --cobertura --output-path results.xml` but the error occurs with any format:

```
No filenames specified!
error: failed to generate report: process didn't exit successfully: `/home/user/.msrustup/packages/rust.tools-x86_64-unknown-linux-gnu.1.65.0-ms-20221105.1/tools/lib/rustlib/x86_64-unknown-linux-gnu/bin/llvm-cov export -format=text -instr-profile=/home/user/projects/my-project/target/llvm-cov-target/my-project.profdata -ignore-filename-regex '/rustc/[0-9a-f]+/|^/home/user/projects/my\-project(/.*)?/(tests|examples|benches)/|^/home/user/projects/my\-project/target/llvm\-cov\-target($|/)|^/home/user/\.cargo/(registry|git)/|^/home/user/\.rustup/toolchains($|/)'` (exit status: 1)
```

**Comment by taiki-e** [comment]
Hmm, `cargo llvm-cov report` couldn't seem to find the object file. What commands did you run before `cargo llvm-cov report`?

**Comment by aaronenberg** [comment]
`cargo llvm-cov —no-report`

**Comment by taiki-e** [comment]
Hmm. I wonder if something odd is going on since that command should be generating object files. Information on the project where the problem occurred or an example to reproduce the problem would be helpful.

**Comment by aaronenberg** [comment]
Luckily I found the commits where the issue was introduced.

The issue is caused by adding this to Cargo.toml:

[[bin]]
name = "my-bin"
path = "src/main.rs"

**Comment by taiki-e** [comment]
Thanks for the repro! Filed #232 to fix this.

**Comment by taiki-e** [comment]
Fixed in v0.5.3. Thanks again for the report!
